lib/vdsm/storage/fileVolume.py

```diff
@@ -580,9 +579,7 @@ def _allocate_volume(cls, vol_path, size, preallocate):
                             level=logging.INFO,
                             log=cls.log):
                         op.run()
-        except exception.ActionStopped:
-            raise
-        except Exception as e:
+        except cmdutils.Error as e:
```

Comment:
This is little bit more fragile - if code in qemuimg.create() or fallocate() changes and another error will be raised this check will fail, but I think this is better than the current code.